### ISSUE TITLE
Data update and then scroll, does not show new data in formatted column

### ISSUE THREAD

**Issue by ManasSinha** [adazzle/react-data-grid#1262]
The grid is populated and on a page event the data for the grid updated. now the visible rows are updated with new value. But when I scroll down and new rows come into view-port, they are not updated with new value. But when I click on that new row, it gets populated with new row.

Upon investigation I found that rowgetter is executing,  but the formatter for that row and collumn is not executing until I click. Once the formatter is executed, the value updates. 

So basically rowgetter is not executing the formatter render() automatically. That is why the column that contain a slider is not updated with new value. but the things that are already in view port are updated. 

Is there any way already implemented to trigger this update.
